modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -491,6 +493,8 @@ public void processOpts() {
                     additionalProperties.put(SINGLE_CONTENT_TYPES, "true");
                     this.setSingleContentTypes(true);
                 }
+                additionalProperties.put(USE_FEIGN_CLIENT, "true");
```

Comment:
Ah I guess this didn't exist, it just relied on importing the correct `apiClient.mustache` instead

Reply:
Yes correct